integration-cli/docker_cli_rm_test.go

```diff
@@ -31,9 +31,9 @@ func (s *DockerSuite) TestRmRunningContainer(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	createRunningContainer(c, "foo")
 
-	if _, _, err := dockerCmdWithError("rm", "foo"); err == nil {
-		c.Fatalf("Expected error, can't rm a running container")
-	}
+	out, _, err := dockerCmdWithError("rm", "foo")
+	c.Assert(err, check.NotNil, check.Commentf(out))
```

Comment:
You should keep the comment, this would give :
```go
c.Assert(err, checker.NotNil, check.Commentf("Expected error, can't rm a running container"))
```